### PR TITLE
AV-191743 doc changes

### PR DESCRIPTION
**Pull request by arihantg**

### CODE REVIEW

**Comment by arihantg** [comment]
build amko

File: docs/crds/gslbconfig.md [hunk 1]
```diff
@@ -36,4 +36,9 @@ spec:
 ### Notes
 * Only one `GSLBConfig` object is allowed.
 * If using `helm install`, a `GSLBConfig` object is created by picking up values from the `values.yaml` file.
-* During `helm delete`, the `GSLBConfig` that holds the UUID of the current instance is deleted. Hence a cleanup of stale GSLB services, if any, is required at the controller before re-installing AMKO.
+* During `helm delete`, the `GSLBConfig` that holds the UUID of the current instance is deleted. To maintain the correct state of AMKO when you install AMKO again conserve the amkoUUID from annotations of GSLBconfig and add it in `configs.amkoUUID` field of [values.yaml](../../README.md#parameters). Otherwise a cleanup of stale GSLB services, if any, is required at the controller before re-installing AMKO.
```
Comment (akshayhavile) [inline docs/crds/gslbconfig.md:39]: should we provide restoring of amkoUUID as step by step guide like `take backup of amkouuid,  use that amkouuid in values.yaml` ?

File: README.md [hunk 2]
```diff
@@ -191,6 +191,7 @@ helm upgrade amko-1598451370 oci://projects.registry.vmware.com/ako/helm-charts/
 | `configs.federation.memberClusters`            | member clusters on which federation should be done                                                                                    |  |
 | `configs.gslbLeaderController`                         | GSLB leader site URL                                                                                                     | Nil                                   |
 | `configs.tenant`                         | GSLB leader controller tenant                                                                                                   | `admin`                                    |
+| `configs.amkoUUID`                         | Configure AMKO with a unique UUID in standard form of xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx with hex-digits.Should be used to recover AMKO state if [GSLBConfig](docs/crds/gslbconfig.md) is deleted                                                                                                  | AMKO will self-generate if left empty                                    |
```
Comment (akshayhavile) [inline README.md:194]: should we re-word it as `Should be used to recover AMKO state if [GSLBConfig](docs/crds/gslbconfig.md) is deleted. AMKO will self-generate it if left empty` ?

File: docs/crds/gslbconfig.md (hunk 1 above)
Comment (arihantg) [reply docs/crds/gslbconfig.md:39]: I think this should be kept here . I will also add a detailed guide in troubleshooting section

File: README.md (hunk 2 above)
Comment (arihantg) [reply README.md]: done

**Review by akshayhavile — APPROVED** [review]

**Pull request merged**